src/metaopt/algo/base.py

```diff
@@ -90,8 +90,8 @@ def suggest(self, num=1):
 
     @abstractmethod
     def observe(self, points, results):
-        """Observe evaluation `results` corresponding to list of `points` in
-        space.
+        """Observe the `results` of the evaluation of the `points` in the
```

Comment:
I would add a similar note as for `judge` and make clear the difference between the two methods.